util/git/client.go

```diff
@@ -440,8 +443,14 @@ func (m *nativeGitClient) LsFiles(path string, enableNewGitFileGlobbing bool) ([
 			if err != nil {
 				return nil, err
 			}
-			if strings.HasPrefix(absPath, m.root) {
-				files = append(files, file)
+
+			if strings.HasPrefix(absPath, realRoot) {
+				// removing the repository root prefix from the file path
+				relativeFile, err := filepath.Rel(realRoot, file)
+				if err != nil {
+					return nil, err
+				}
+				files = append(files, relativeFile)
 			} else {
 				log.Warnf("Absolute path for %s is outside of repository, removing it", file)
```

Comment:
I know this doesnt affect the change but should we update this log message to ignoring it? It feels misleading to say "removing". 
```suggestion
				log.Warnf("Absolute path for %s is outside of repository, ignoring it", file)
```

Reply:
I agree